tests/units/Migration.php

```diff
@@ -159,7 +162,24 @@ function () {
          'value'     => 'setted value'
       ]];
       $it = new \ArrayIterator($dbresult);
-      $this->calling($this->db)->request = $it;
+      $this->db = new \mock\DB();
```

Comment:
I do not understand this change.

Reply:
I need to reinstanciate the mock to to mock the request() method. See a few lines below:
```php
$this->calling($this->db)->request[1] = $it;
$this->calling($this->db)->request[2] = [];
```
It is hard to know how namy calls to request() were done before reaching this scenario. The purpose is to reset the call counter of the method and be a bit more robust if later the count of calls changes as side effect of future development.

Reply:
Stilll do not understand... `request` was called once; it is now called twice, and second is empty? 
Your changes should not affect Migration class; something sound not correct (or I just do not get the point) :/